From 23d6eaf944e7f358e18a24778a5cc2bc4df4a109 Mon Sep 17 00:00:00 2001
From: Kees Cook <keescook@chromium.org>
Date: Tue, 2 Oct 2012 11:18:05 -0700
Subject: [PATCH 650/858] drivers/net/ethernet/intel: remove depends on
 CONFIG_EXPERIMENTAL

The CONFIG_EXPERIMENTAL config item has not carried much meaning for a
while now and is almost always enabled by default. As agreed during the
Linux kernel summit, remove it from any "depends on" lines in Kconfigs.

CC: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
CC: Jesse Brandeburg <jesse.brandeburg@intel.com>
CC: Bruce Allan <bruce.w.allan@intel.com>
CC: Carolyn Wyborny <carolyn.wyborny@intel.com>
CC: Don Skidmore <donald.c.skidmore@intel.com>
CC: Greg Rose <gregory.v.rose@intel.com>
CC: Peter P Waskiewicz Jr <peter.p.waskiewicz.jr@intel.com>
CC: Alex Duyck <alexander.h.duyck@intel.com>
CC: John Ronciak <john.ronciak@intel.com>
Signed-off-by: Kees Cook <keescook@chromium.org>
Acked-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
(cherry picked from commit 81852c2941fbc321343c5c2523f0916aa438e3ed)

Signed-off-by: Arnaldo Carvalho de Melo <acme@redhat.com>

Conflicts:
 drivers/net/ethernet/intel/Kconfig
---
 drivers/net/ethernet/intel/Kconfig | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/net/ethernet/intel/Kconfig b/drivers/net/ethernet/intel/Kconfig
index bde4f3d..6c78633 100644
--- a/drivers/net/ethernet/intel/Kconfig
+++ b/drivers/net/ethernet/intel/Kconfig
@@ -5,11 +5,6 @@
 config NET_VENDOR_INTEL
  bool "Intel devices"
  default y
- depends on PCI || PCI_MSI || ISA || ISA_DMA_API || ARM || \
-     ARCH_ACORN || SNI_RM || SUN3 || \
-     GSC || BVME6000 || MVME16x || \
-     (ARM && ARCH_IXP4XX && IXP4XX_NPE && IXP4XX_QMGR) || \
-     EXPERIMENTAL
  ---help---
    If you have a network (Ethernet) card belonging to this class, say Y
    and read the Ethernet-HOWTO, available from
-- 
1.8.3.1